### ISSUE TITLE
Error with Yoast sitemap

### ISSUE THREAD

**Issue by strarsis** [linchpin/mesh#116]
When opening sitemap.xml from Yoast, request fails with:
````
[...]
#4 /srv/www/example.com/releases/20170421200153/web/app/plugins/wordpress-seo/inc/sitemaps/class-tax" while reading response header from upstream, client: <client ip>, server: www.example.com, request: "GET /sitemap_index.xml HTTP/2.0", upstream: "fastcgi://unix:/var/run/php-fpm-wordpress.sock:", host: "www.example.com"
2017/04/22 13:10:03 [error] 1420#1420: *127944 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Mesh::wpseo_sitemap_exclude_taxonomy(), 1 passed in /srv/www/example.com/releases/20170421200153/web/wp/wp-includes/class-wp-hook.php on line 300 and exactly 2 expected in /srv/www/example.com/releases/20170421200153/web/app/plugins/mesh/class.mesh.php:1012
Stack trace:
#0 /srv/www/example.com/releases/20170421200153/web/wp/wp-includes/class-wp-hook.php(300): Mesh->wpseo_sitemap_exclude_taxonomy(false)
#1 /srv/www/example.com/releases/20170421200153/web/wp/wp-includes/plugin.php(203): WP_Hook->apply_filters(false, Array)
#2 /srv/www/example.com/releases/20170421200153/web/app/plugins/wordpress-seo/inc/sitemaps/class-taxonomy-sitemap-provider.php(270): apply_filters('wpseo_sitemap_e...', false, 'custom-taxonomy')
#3 [internal function]: WPSEO_Taxonomy_Sitemap_Provider->is_valid_taxonomy('custom-taxonomy')
#4 /srv/www/example.com/releases/20170421200153/web/app/plugins/wordpress-seo/inc/sitemaps/class-tax" while reading response header from upstream, client: <client ip>, server: www.example.com, request: "GET /sitemap_index.xml HTTP/2.0", upstream: "fastcgi://unix:/var/run/php-fpm-wordpress.sock:", host: "www.example.com"
[...]
````

````
[...]
Uncaught ArgumentCountError: Too few arguments to function Mesh::wpseo_sitemap_exclude_taxonomy(), 1 passed in 
[...]
````

**Comment by aaronware** [comment]
@strarsis do you have any specs on the PHP version etc of the env you're using? We have it running on a few env 5.6 to 7.1 using Yoast and haven't seen that issue. I just looked at a few live sites as well without issue.

https://linchpin.agency/sitemap_index.xml for example.

I can definitely put in a default value for the 2nd args. Just wanted to see if it was a specific php version or setting.

**Comment by strarsis** [comment]
PHP version is PHP 7.1.4-1+deb.sury.org~xenial+1.
WordPress [Bedrock](https://roots.io/bedrock/) running on [Trellis](https://roots.io/trellis/).

**Comment by aaronware** [comment]
@nate-allen gonna toss this one your way if you can take a look when you are working through your other features. Might be something we can hotfix in our next release.

**Comment by aaronware** [comment]
@strarsis This is fixed and will be merged into the next release. Apparently I was missing the argument count in the add_filter declaration so template name was never being passed. https://github.com/Yoast/wordpress-seo/issues/7026

**Comment by aaronware** [comment]
Fixed and pushed to develop for 1.2 release
